src/train.js

```diff
@@ -74,16 +72,18 @@ const extractTrainingLabel = (state, row) => {
   return convertValueForTraining(state, state.labelColumn, row);
 };
 
-const prepareTrainingData = () => {
+const prepareTrainingData = store => {
   const updatedState = store.getState();
-  const trainingExamples = updatedState.data
-    .map(row => extractTrainingExamples(updatedState, row));
-  const trainingLabels = updatedState.data
-    .map(row => extractTrainingLabel(updatedState, row));
+  const trainingExamples = updatedState.data.map(row =>
+    extractTrainingExamples(updatedState, row)
+  );
+  const trainingLabels = updatedState.data.map(row =>
+    extractTrainingLabel(updatedState, row)
+  );
   /*
   Select X% of examples and corresponding labels from the training set to use for a post-training accuracy calculation.
   */
-  const numToReserve = parseInt(
+  const numToReserve = Math.ceil(
```

Comment:
@Erin007 Actually, will we always want to round up?

Reply:
I can't think of a time when we wouldn't want to round up.